Atomic/sign.py

```diff
@@ -25,24 +30,38 @@ def cli(subparser):
                        help=_("Define an alternate directory to store signatures"))
 
 class Sign(Atomic):
-    def sign(self):
+    def __init__(self):
+        super(Sign, self).__init__()
+
+    def sign(self, debug=None, signer=None, in_signature_path=None, images=None):
```

Comment:
@rhatdan I will push a new update here shortly.  I have removed all the named args that overlap with self.args between push and sign.  That only leaves images and in_signature_path which are not in push.